README.md

```diff
@@ -15,9 +15,9 @@
 
 <div align="center">
 
-| Travis                                                                                                                 | Deb                                           | RPM                                           | War                                           |
+| Release                                                                                                                 | Deb                                           | RPM                                           | War                                           |
 | ---------------------------------------------------------------------------------------------------------------------- | --------------------------------------------- | --------------------------------------------- | --------------------------------------------- |
-| [![Travis CI](https://travis-ci.org/rundeck/rundeck.svg?branch=master)](https://travis-ci.org/rundeck/rundeck/builds#) | [Download](https://www.rundeck.com/downloads) | [Download](https://www.rundeck.com/downloads) | [Download](https://www.rundeck.com/downloads) |
+| [![Release Build](https://img.shields.io/github/actions/workflow/status/rundeck/rundeck/release.yml)](https://github.com/rundeck/rundeck/actions/workflows/release.yml) | [Download](https://www.rundeck.com/downloads) | [Download](https://www.rundeck.com/downloads) | [Download](https://www.rundeck.com/downloads) |
```

Comment:
to show CI status, you should use a CircleCI badge: https://img.shields.io/circleci/build/github/rundeck/rundeck/main?logo=circleci

Reply:
Does CircleCI have a public page to display build statuses for Rundeck? Can't seem to find one, only GH Actions page. Happy to include just the badge without the link to the page, though it would be informative if users could check the build statuses. Thanks!

Reply:
They don't have a public dashboard apparently.  We don't use GH actions extensively.  Since it's not public, it may not be useful to display the badge either.

Reply:
Ok thanks, badge is removed.